modules/kubefed-registering-clusters.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+// * kubefed/kubefed-resource-federation.adoc
+
+[id="kubefed-registering-clusters_{context}"]
+= Registering (joining) clusters
+
+The `join` operation reads information about how to connect the registering cluster and the cluster hosting federation from your local `kubeconfig` file.
+
+You can use the `kubefedctl join` command to register clusters for {KubeFedProductShortName} resource federation.
+
+.Procedure
+
+. Use the `kubefedctl join` command with the required arguments as shown in the example.
++
+----
+$ kubefedctl join <cluster-name> --cluster-context <cluster-context> \
```

Comment:
I think it would be useful for the end-users to understand different flags used in this command if we provide some information around it. Something similar to [this](https://github.com/openshift/kubefed-operator/blob/master/deploy/olm-catalog/kubefed-operator/4.2/kubefed-operator.v4.2.0.clusterserviceversion.yaml#L419-423)